udata/tasks.py

```diff
@@ -78,6 +108,19 @@ def log_test(self):
     self.log.error('This is a ERROR message')
 
 
+@job('test-low-queue', queue='low')
+@job('test-default-queue', queue='default')
+@job('test-high-queue', queue='high')
+def queue_test(self, raises=False):
```

Comment:
can't be defined in test files?

Reply:
No, this is intentional, like `log-test` and `error-test`: this allow to manually run these jobs with `udata job run -d test-high-queue` and see if this is properly routing.
You can also run `udata job run -d test-high-queue raises=True` and you can have the time for a tasks to be processed because the exception will appear in Sentry with a timestamp.